R/boxr_source.R

```diff
@@ -1,17 +1,16 @@
 #' Source R code from a Box file
 #' 
-#' `box_source` will download a remote R script stored on box.com, and then
-#' attempt to execute it, using [source()].
+#' This function downloads a file from Box, then runs its 
+#' contents, as R code, using [source()].
 #' 
 #' @inheritParams box_dl
+#' @return Object returned by [source()].
```

Comment:
I've always assumed this function doesn't return anything it just populated the environment as it executes the sourced file

Reply:
I just had to check myself - it returns a list, invisibly, of the last value in the sourced file:

something like


```r
str(source("tmp.R"))
```

```
List of 2
 $ value  : num 2
 $ visible: logi FALSE
```

`tmp.R` contains:

```r
tmp <- 3
b <- 2
```

----

That being said, I'll make sure that we say that this is *just* like source, and will likely change your environment.

Reply:
I ended up modifying the `box_source()` function a bit, so that it accepts `...` to pass on to `source()`.